gitstats

```diff
@@ -389,6 +390,11 @@ class GitDataCollector(DataCollector):
 
             sha1, fullpath = line.split()[2:4]
 
+            # skip directories we aren't interested in
+            if len(conf['exclude_directories']):
+                if any(dir in fullpath for dir in conf['exclude_directories'].split(',')):
```

Comment:
Thank you for the comprehensive feedback.  I will make a few changes based on this and commit soon.

BTW: any idea on how I can test the results of my changes relatively effectively and consistently?  I can run a report against my own fairly comprehensive repos, but not sure that I can easily determine that the results are correct.